### PR TITLE
chore: turn off talisman for ephemeral environments in ci

### PR DESCRIPTION
**Pull request by eschutho**
### SUMMARY
Talisman is throwing a lot of blocking script-src errors in the ephemeral testing environment for CI, and it's blocking login to the application in CI. I'm disabling Talisman since the CSP rules aren't needed there, unless for testing. 

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by craig-rueda — APPROVED** [review]

File: superset/config.py [hunk 1]
```diff
@@ -1397,6 +1397,7 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
     },
     "content_security_policy_nonce_in": ["script-src"],
     "force_https": False,
+    "session_cookie_samesite": "None",
```
Comment (eschutho) [inline superset/config.py:1400]: just testing if this works.. the long term solution would be to put this in the github workflow above.

**Comment by eschutho** [comment]
@michael-s-molina we had talked about solving config issues for CSP so that we can keep it on in ephemerals. Since we've been blocked on using ephemerals for a bit now, I'm leaning towards merging this to turn off CSP and then coming back shortly afterward to test turning it back on since it could take longer to solve. WDYT?

**Comment by michael-s-molina** [comment]
> @michael-s-molina we had talked about solving config issues for CSP so that we can keep it on in ephemerals. Since we've been blocked on using ephemerals for a bit now, I'm leaning towards merging this to turn off CSP and then coming back shortly afterward to test turning it back on since it could take longer to solve. WDYT?

Sounds good 👍🏼

**Comment by michael-s-molina** [comment]
@eschutho Could you rebase this PR to fix the Docker issue?

**Pull request merged**